crates/tabby-download/Cargo.toml

```diff
@@ -9,4 +9,4 @@ anyhow = { workspace = true }
 tracing = { workspace = true }
 tokio-retry = "0.3.0"
 sha256 = "1.4.0"
-aim = "1.8.5"
+#aim = "1.8.5"
```

Comment:
Failed to compile on windows platform.

After tracing, I found dependence chain:

```
aim -> skim-navi -> skim -> tuikit -> nix
```

This `nix` crate is specifically for *nix platform